src/components/webviews/jsInteractions/jsCozyInjection.ts

```diff
@@ -17,6 +17,7 @@ const makeMetadata = (routeName?: string): string => {
   return JSON.stringify({
     immersive: routeName ? immersiveRoutes.includes(routeName) : false,
     navbarHeight,
+    offline_available: true,
```

Comment:
Why is it the best way to announce that offline is available ?

Reply:
Not sure if this is the best way, but it is the easiest way I found for now. This is because we should check for offline compatibility during the cozy-app initialization, and in some cozy-apps we don't have access to cozy-intent yet during this phase. So injecting it here is easier to implement than using cozy-intent.